src/setup/cortx_setup.py

```diff
@@ -66,15 +68,17 @@ def add_args(parser: str):
 
     def process(self, *args, **kwargs):
         """ Bootsrap Cluster """
-        if self._args.action == "bootstrap":
-            CortxProvisioner.cluster_bootstrap(self._args.node_id, self._args.cortx_conf)
+        if self._args.action == 'bootstrap':
```

Comment:
For consistency with config apply, lets add -f command option.

Reply:
Done